Tasks/UseDotNetV2/README.md

```diff
@@ -24,6 +24,7 @@ Please report a problem at [Developer Community Forum](https://developercommunit
 
 * **Package to install\*:** You can choose to install either runtime or SDK.
 
+* **Use global json**: This checkbox indicates that all versions from all `global.json` files will be used to install the sdk versions. You can set the search root path with `Working Directory`.
```

Comment:
`Working directory` should also be there in `Parameters of the task` section here.